### PR TITLE
Add template files to phpstan checks

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

Check template files with phpstan.

- started with +16k errors
- made some regex replacements
- baseline 11k errors

Yeah, 11k baseline is not nice, but that will be reduced during days. (help welcome to add doc blocks to template files!)

### Related Pull Requests

1. See OpenMage/magento-lts#2847

### CODE REVIEW

**Comment by ADDISON74** [comment]
This is a very good idea, but it will involve even more effort considering the large number of files that require changes.

**Comment by sreichel** [comment]
There is no more effort.

Having it baselined makes it easier to track changes.

This is a blindly-hit-approved.

File: app/design/adminhtml/default/default/template/downloadable/product/edit/downloadable/links.phtml [hunk 1]
```diff
@@ -17,7 +17,8 @@
 
 /**
  * @see Mage_Downloadable_Block_Adminhtml_Catalog_Product_Edit_Tab_Downloadable_Links
- * @var $this Mage_Downloadable_Block_Adminhtml_Catalog_Product_Edit_Tab_Downloadable_Links
+ * @var Mage_Downloadable_Block_Adminhtml_Catalog_Product_Edit_Tab_Downloadable_Links $this
+ * @var Mage_Downloadable_Block_Adminhtml_Catalog_Product_Edit_Tab_Downloadable_Links $this
```
Comment (ADDISON74) [inline app/design/adminhtml/default/default/template/downloadable/product/edit/downloadable/links.phtml:21]: The line seems to be doubled. If it is an error there are more places for changes.

**Review by ADDISON74 — CHANGES_REQUESTED** [review]

**Review by ADDISON74 — APPROVED** [review]

**Review by ADDISON74 — APPROVED** [review]

File: app/design/adminhtml/default/default/template/bundle/sales/creditmemo/create/items/renderer.phtml [hunk 2]
```diff
@@ -9,13 +9,14 @@
...
  * @package     default_default
  * @copyright   Copyright (c) 2006-2020 Magento, Inc. (https://www.magento.com)
- * @copyright   Copyright (c) 2018-2022 The OpenMage Contributors (https://www.openmage.org)
+ * @copyright   Copyright (c) 2018-2024 The OpenMage Contributors (https://www.openmage.org)
  * @license     https://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
  */
 ?>
 <?php
 /**
  * @see Mage_Bundle_Block_Adminhtml_Sales_Order_Items_Renderer
```
Comment (kiatng) [inline app/design/adminhtml/default/default/template/bundle/sales/creditmemo/create/items/renderer.phtml:18]: Possible to remove line 18?

**Comment by ADDISON74** [comment]
There is an error in a template file reported recently:

**Ignored error pattern #^Variable \$this might not be defined\.$# in path /home/runner/work/magento-lts/magento-lts/app/design/adminhtml/default/default/template/customer/tab/view.phtml was not matched in reported errors.**

File: app/design/adminhtml/default/default/template/bundle/sales/creditmemo/create/items/renderer.phtml (hunk 2 above)
Comment (sreichel) [reply app/design/adminhtml/default/default/template/bundle/sales/creditmemo/create/items/renderer.phtml:18]: see annotation is in lot of files and its used for phpdocumentor. I'd keep.

Comment (kiatng) [reply app/design/adminhtml/default/default/template/bundle/sales/creditmemo/create/items/renderer.phtml:18]: How about consistency? The `see` annotation is also not in a lot of files. For consistency, is it better to remove?

**Review by ADDISON74 — APPROVED** [review]

**Review by kiatng — DISMISSED** [review]

File: app/design/adminhtml/default/default/template/bundle/sales/creditmemo/create/items/renderer.phtml (hunk 2 above)
Comment (sreichel) [reply app/design/adminhtml/default/default/template/bundle/sales/creditmemo/create/items/renderer.phtml:18]: "See" can have different or multiple values.

I am not going to add them when not helpfull, but i'd keep the existing.

**Review by kiatng — APPROVED** [review]

**Review by ADDISON74 — APPROVED** [review]

**Pull request merged**